packages/wasm/src/helper.ts

```diff
@@ -0,0 +1,55 @@
+export const HELPERS_ID = '\0wasmHelpers.js';
```

Comment:
What does this null byte prefix do?

Reply:
I am not 100 percent sure where to find the documentation, but "\0" seems to be a convention when writing rollup plugin to indicate the module comes from a plugin. and if it should not be taken by this plugin, resolveId returns null, otherwise returns the current resolveId

Reply:
It indicates a "virtual module" in the Rollup ecosystem.

Reply:
More precisely it means: Other plugins, please ignore this module and please do not apply any transformations etc.  https://rollupjs.org/guide/en/#conventions